pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -116,6 +116,7 @@ Activity:
     - lastUpdatedText
     - lastUpdated
     - isFollowup
+    - isHidden
```

Comment:
@fat23cat @bskinner We're adding new properties to APIs. There's actually two different flags for "hidden" but I don't think that's a detail client needs to know, so I settled on one `isHidden` flag.

Reply:
What is the expected behavior if the flag is set?

Reply:
I agree that internal to the server, the two flags are okay (although kind of confusing) but that client should one `isHidden`.  @yufengwng are we only including an activity if `isHidden` is true _and_ the user has admin access?  If not, we should do that.  Otherwise we are leaking an activity and asking the client to hide it.  So my two cents @bskinner is that the mobile client should ignore this for now because we haven't thought through the study staff UX when using the masquerade-type feature that this is used for in the prism route (currently only for testboston).

Reply:
I shall answer my own question: we are filtering things appropriately (`filterActivityInstancesFromDisplay()`)

Reply:
So @bskinner I think it's safe to take no action client-wise, but I think the intended behavior here should be a comment in the schema to the effect of "If true, client should indicate to the user that this activity is not visible to the participant, still make it visible"?

Reply:
That seems contradictory on its face- Is that from the perspective of the operator?

Reply:
Re: the two flags. So this evolved by design. One flag hides a single instance, while the other flag hides all instances of an activity regardless of the other flag, so it gives us control over hiding.

Re: expected behavior. `isHidden` means whether activity was hidden _from the participant_. It relates to the participant, which is my implicit understanding but I should have put that in words in the documentation. If flag is set, then as Zim mentioned client should prefer to show some indicator that it's hidden _from the participant_. Sometimes, such as in the case of study admin, the operator is not the participant, so it may be hidden from participant but not from operator.

Reply:
Ok, so let me parity check my understanding:

* If `isHidden` is `false`, the participant _may_ see the instance, or activity, as normal.
* If `isHidden` is `true`, the participant _may not_ see the instance, or activity. Users with management privileges _may_ see the participants instances, or activites, but should have a visible indicator they are hidden from the participant.

Sound about right?